src/Microsoft.ML.PCA/PCACatalog.cs

```diff
@@ -15,21 +15,21 @@ public static class PcaCatalog
         /// <param name="catalog">The transform's catalog.</param>
         /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
         /// <param name="inputColumnName">Name of column to transform. If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.</param>
-        /// <param name="weightColumn">The name of the weight column.</param>
+        /// <param name="exampleWeightColumnName">The name of the example weight column (optional).</param>
         /// <param name="rank">The number of principal components.</param>
         /// <param name="overSampling">Oversampling parameter for randomized PrincipalComponentAnalysis training.</param>
         /// <param name="center">If enabled, data is centered to be zero mean.</param>
         /// <param name="seed">The seed for random number generation.</param>
         public static PrincipalComponentAnalysisEstimator ProjectToPrincipalComponents(this TransformsCatalog.ProjectionTransforms catalog,
             string outputColumnName,
             string inputColumnName = null,
-            string weightColumn = PrincipalComponentAnalysisEstimator.Defaults.WeightColumn,
+            string exampleWeightColumnName = PrincipalComponentAnalysisEstimator.Defaults.WeightColumn,
```

Comment:
Defaulting this to `null` will be covered in #1065, right?

Reply:
Yes, perhaps #1065 would be the right place to  set this to `null` as default value, since that issue related to changing the _behavior_ of the weights parameters.

The issues addressed by this PR are all related  to _consistency_ in API naming convention.

Reply:
#1065 did not address this. hence taking care of this in this PR

---
In reply to: [259028009](https://github.com/dotnet/machinelearning/pull/2665#discussion_r259028009) [](ancestors = 259028009)

Reply:
fixed in this PR itself.

---
In reply to: [259039000](https://github.com/dotnet/machinelearning/pull/2665#discussion_r259039000) [](ancestors = 259039000)